packages/edgeworth/tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
+    "suppressImplicitAnyIndexErrors": true,
```

Comment:
What's the dealio here?

Reply:
This is in `core` and `automator` as well so I copied them over. We shouldn't really have this turned on tbh

Reply:
https://github.com/penrose/penrose/issues/1370